src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/ChestProfit.kt

```diff
@@ -136,7 +164,7 @@ object ChestProfit {
             val enchant = name.substring(name.indexOf("(") + 1, name.indexOf(")"))
             return enchantNameToID(enchant)
         } else {
-            val unformatted = name.stripControlCodes()
+            val unformatted = name.stripControlCodes().replace("Shiny ", "")
```

Comment:
not sure about this one

Reply:
unless there's a different way of doing it, they currently show up as 0 coins in the profit overview in croesus (without clicking into the chest individually)

Reply:
actually its dungeons only was worried that they might add an item with shiny in the name